src/hotspot/share/services/attachListener.hpp

```diff
@@ -59,6 +60,20 @@ enum AttachListenerState {
   AL_INITIALIZED
 };
 
+/*
+Version 1 (since jdk6): attach operations always have 3 (AttachOparation::arg_count_max)
+  arguments, each up to 1024 (AttachOparation::arg_length_max) symbols.
```

Comment:
"AttachOparation" typo and also "symbols" is clarified to mean characters in a review comment, so should probably change that here and also in attachListener.cpp 626, 627.
CompatTest.java says "1024 symbols"

Reply:
Fixed.